x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -237,7 +237,11 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
               settings: {
                 number_of_shards: 1,
               },
-              mappings: {}, // TODO: Add mappings here via `mappingFromFieldMap()`
+              mappings: mappingFromFieldMap({
```

Comment:
Temporary workaround to unblock work, will not be merged. Seems to be newly introduced mapping issues related to the security solution rules that are being worked on.